java/elemental2/chrome/chrome.js.diff

```diff
@@ -0,0 +1,36 @@
+295c295
+<  * @extends {ChromeBaseEvent<function()>}
+---
+>  * @extends {ChromeBaseEvent<!Function>}
+303c303
+<  * @extends {ChromeBaseEvent<function(string)>}
+---
+>  * @extends {ChromeBaseEvent<!Function>}
+311c311
+<  * @extends {ChromeBaseEvent<function(boolean)>}
+---
+>  * @extends {ChromeBaseEvent<!Function>}
+319c319
+<  * @extends {ChromeBaseEvent<function(number)>}
+---
+>  * @extends {ChromeBaseEvent<!Function>}
+327c327
+<  * @extends {ChromeBaseEvent<function(!Object)>}
+---
+>  * @extends {ChromeBaseEvent<!Function>}
+335c335
+<  * @extends {ChromeBaseEvent<function(!Array<string>)>}
```

Comment:
the build was having issues with these signatures, so i've made them generic for now, until they can get better type visibility

Reply:
You should try to fix that in https://github.com/google/closure-compiler/blob/master/contrib/externs/chrome.js directly.